WBooks/Detail View/DetailViewController.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  DetailViewController.swift
+//  WBooks
+//
+//  Created by Gaston Maspero on 30/05/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+class DetailViewController: UIViewController {
+    
+    private lazy var _view: DetailView = DetailView.loadFromNib()!
+    
+    private var _viewModel: DetailViewModel
+    
+    init(viewModel: DetailViewModel) {
+        _viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    override var preferredStatusBarStyle: UIStatusBarStyle {
+        return .lightContent
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func loadView() {
+        view = _view
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setNeedsStatusBarAppearanceUpdate()
+        setBookDetails()
+        addNavBarButtons()
+    }
+    
+    private func setBookDetails() {
+        _view.setBook(bookViewModel: _viewModel.bookViewModel)
+        _viewModel.loadRents(onSuccess: onLoadRentsSuccess)
+    }
+    
+    func onLoadRentsSuccess(rents: [Rent]) {
+        _viewModel.onLoadRentsSuccess(rents: rents)
+        _view.setAvailability(status: _viewModel.status)
+        _view.rent.addTarget(self, action: #selector(rent), for: .touchUpInside)
+    }
+    
+    @objc func rent() {
+        switch _viewModel.status {
+        case .available:
+            let alert = UIAlertController(alertViewModel: ErrorAlertViewModel(title: "THANKS".localized(), message: "PROCESSING_RENT".localized(), dismissButtonTitle: "ACCEPT".localized()))
+            self.present(alert, animated: true, completion: nil)
+            _viewModel.rentBook(onSuccess: onBookRentSuccess, onError: onBookRentError)
+        case .notAvailable, .inHands:
+            let alert = UIAlertController(alertViewModel: ErrorAlertViewModel(title: "UPS!", message: "CANNOT_RENT".localized(), dismissButtonTitle: "ACCEPT".localized()))
+            self.present(alert, animated: true, completion: nil)
+        case .notLoaded:
+            let alert = UIAlertController(alertViewModel: ErrorAlertViewModel(title: "UPS!", message: "There seems to be a problem. Please try again later.", dismissButtonTitle: "ACCEPT".localized()))
```

Comment:
Is there any reason why this string is not in Localizable?

Reply:
It slipped my mind. Changed it!